lib/media/helper_modules/helpers.ex

```diff
@@ -578,32 +578,28 @@ defmodule Media.Helpers do
   end
 
   def delete_files(files_to_delete) do
-    unless test_mode?(),
```

Comment:
why removing the unless condition ?

This means it shouldn't be running in test mode which makes sense (do not hit S3 server if you are not in real mode)

Reply:
Tests already cover this case by using a mock for the functions (this calls delete_file and delete_file is mocked)